### PR TITLE
don't re-serialize the whole message when headers have been prepended but nothing else has changed

### PR DESCRIPTION
**Pull request by jeremyschlatter**

### CODE REVIEW

**Comment by jeremyschlatter** [comment]
Replaces #83

File: tests/mime/message/part_test.py [hunk 1]
```diff
@@ -222,6 +222,19 @@ def preserve_ascii_test():
     eq_('7bit', message.content_encoding.value)
 
 
+def preserve_formatting_with_new_headers_test():
+    """Make sure that we don't re-serialize a message and change its formatting when headers were added but nothing else was modified."""
```
Comment (jessespears) [inline tests/mime/message/part_test.py]: Please break this line

**Comment by jessespears** [comment]
lgtm

File: tests/mime/message/part_test.py (hunk 1 above)
Comment (jeremyschlatter) [inline tests/mime/message/part_test.py]: Done.

**Pull request merged**